projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk-jupyterlab-extension/ui_job_properties.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2023-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from enum import Enum
+
+
+class JobProperties(Enum):
```

Comment:
maybe VdkOptions, due restApiUrl is not a job property, and the file name also
```suggestion
class VdkOption(Enum):
```

Reply:
Done